build.js

```diff
@@ -11,6 +11,20 @@ const includeFiles = [
   'spinner.css', 'supported_extensions.js',
 ];
 const zipName = './gecko-code-coverage.zip';
+fs.readFile('manifest.json', 'utf8', function(err, data) {
+  var manObj;
+  if (err) throw err;
+  manObj = JSON.parse(data);
+  var packObj;
+  fs.readFile('package.json', 'utf8', function(err, data) {
+      if (err) throw err;
+      packObj = JSON.parse(data);
+      if (manObj["version"] != packObj["version"]) {
+          console.error("Different versions of manifest.json and package.json");
```

Comment:
Let's actually throw an error here, so that the build fails when this happens.
(If we just log the error, the build will finish successfully)